### PR TITLE
Update License Headers and license maven plugin

### PR DESCRIPTION
**Pull request by rafabene**

### CODE REVIEW

**Comment by sgilda** [comment]
Wow! This was a lot of work. Great job!

I do have a question about the kitchesink-angularjs files. They seem to have 2 licenses now. Is that intentional?

**Comment by sgilda** [comment]
I also see modifications to some of the picketlink quickstart files, for example, a license added to the picketlink-authentication-rs-endpoint/src/main/webapp/WEB-INF/faces-config.xml

Those quickstarts are in another repository now.

**Comment by rafabene** [comment]
No it was not. Maybe we should ignore angularjs too. I'll resend the correction

**Comment by rafabene** [comment]
I'll look on authentication-rs-endpoint

**Comment by rafabene** [comment]
Done! PR updated

File: kitchensink-angularjs-topcoat/src/main/webapp/js/libs/mobile-nav.js [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*
  * angular-mobile-nav by Andy Joslin
  * http://github.com/ajoslin/angular-mobile-nav
  * @license MIT License http://goo.gl/Z8Nlo
```
Comment (sgilda) [inline kitchensink-angularjs-topcoat/src/main/webapp/js/libs/mobile-nav.js:4]: This one appears to have 2 licenses. Should it?

**Comment by sgilda** [comment]
Other than the one question above, this looks good to go!

After you answer that, do you want me to sqash and merge?

**Comment by rafabene** [comment]
PR merged

**Pull request closed without merging**